pkg/manager/member/template.go

```diff
@@ -77,6 +77,7 @@ exec /tidb-server ${ARGS}
 `))
 
 type TidbStartScriptModel struct {
+	ClusterName     string
```

Comment:
This is not used, right?

Reply:
Old logic have this field, i think it is unused. I can remove it.